tutorials/spa-extend-outlook-automation/spa-extend-outlook-automation.md

```diff
@@ -0,0 +1,147 @@
+  !![Save and Test](13-TestAutomationExecution.png)
+
+14. You can include additional attributes related to your email in search context result. For ex, see the various activities you can use to retrieve additional attributes in your search context
+  !![Validate the result](14-OtherEmailAttributes.png)
+
+At this stage, we have converted our little project to process a series of emails in search context. An Email may contain one or more file attachments. In the next step, we will download file attachments.
+
+[VALIDATE_1]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Download Attachments from an email)]
+
+In this step, we will create a local folder in your windows file system. We will define folder name dynamically using current timestamp. Then we will save each email attachment to that local folder.
+
+1.  Create a string variable folder name. For this we will be using **String** datatype and drag and drop into the workflow. You can include it above **Get Next Email (Context)** activity as shown below.
+    !![Add Create a String Variable Activity](15-CreateStringVariable.png)
+
+2.  To the string variable, we will assign a timestamp value as its value. this is done using `Date.now().toString()` via the expression editor.
+    > In real world, you may wanted to use your business process specific folder Name
+
+    !![Assign TimeStamp to Variable Name](16-AssignTimeStamptoFolderName.png)
+
+3.  Just make sure, you set desired variable name as the output parameter
+    !![Rename Output Variable](17-StringVariableOutput.png)
+
+4.  Using the folder name we will create a folder in the file system. we will include relevant **Create Folder** activity from the **File System** collection to the flow
+    !![Add Create Folder Activity](18-CreateFolderActivity.png)
+
+5.  In case you wanted the new folder to be created in a specific path, you can append that location to the folder name. make sure you append **double slash** (\\) to the path
+    !![Provide the folder path](19-FolderPath.png)
+
+6.  Here we are planning to save all attachments in an email in one go by adding **Save All Attachments** activity to your flow. You can also individually save every attachment using **Save Mail Attachment**. It is a good idea for you to explore difference between both these options on your own.
+    !![Save All attachments](20-SaveAllAttachments.png)
+
+7.  We will refer the folder path from the previous step as the destination location for saving attachments
+    !![Provide the save location](21-SaveLocation.png)
+
+8.  You can test the project.
+    !![Test the project](22-TestSaveAllAttachments.png)
+
+9.  If any email in your search context contains an attachment, you will see them saved in the local folders. By our logic, you can review it based on recently created folder in your local filesystem
+    !![Validate the result](23-ResultOfSaveAllAttachmentsTest.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Sending an email reply)]
```

Comment:
Use active voice "Send an email reply"

Reply:
ok. fixed and reviewed other tutorials too